vault-harvester/scripts/deploy.ts

```diff
@@ -58,17 +58,40 @@ const deployContract = async (
   contractFactory: ContractFactory,
   constructorArgs: any[]
 ): Promise<{ contract: string; impl: string }> => {
+  if (network.name !== "matic") {
+    console.log("Deploying: MockKeeperRegistry");
+    const MockKeeperRegistryFactory = await ethers.getContractFactory(
+      "MockKeeperRegistry"
+    );
+    const mockKeeperRegistry = await MockKeeperRegistryFactory.deploy();
+    await mockKeeperRegistry.deployed();
+    console.log(`MockKeeperRegistry: ${mockKeeperRegistry.address}`);
+    constructorArgs[0] = mockKeeperRegistry.address;
```

Comment:
Deploying this mock is not necessary because the deploy script requires a deployed `KeeperRegistry` address. It's described in the readme that testing the deployment locally should be done on a forked network - https://github.com/hackbg/chainlink-keepers-templates/tree/main/vault-harvester#deploy.